sportsdataverse/wexp/engines.py

```
@@ -0,0 +1,856 @@
+    # home team the HFA indicator would be constant 1 and collinear with the
+    # unpenalized intercept (singular normal equations); the mirrored away
+    # view makes it vary and pins the intercept ~0 by construction.
+    games = pl.concat(
+        [
+            base.select(
+                "season",
+                "week",
+                off=pl.col("home_team"),
+                deft=pl.col("away_team"),
+                resp=pl.col(resp_col),
+                homeflag=pl.when(pl.col("neutral_site") == True).then(pl.lit("")).otherwise(pl.col("home_team")),
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)